source/transfers/contracts/PartialKittyCoreTransferHandler.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity 0.5.12;
+
+import "./interfaces/ITransferHandler.sol";
+import "./interfaces/IPartialKittyCoreTransferHandler.sol";
+
+contract PartialKittyCoreTransferHandler is ITransferHandler {
+
+  function transferTokens(
+    address _from,
+    address _to,
+    uint256 _param,
+    address _token
+  ) external returns (bool) {
+    IPartialKittyCoreTransferHandler(_token).transferFrom(_from, _to, _param);
```

Comment:
Again `IPartialKittyCoreTransferHandler` is a misleading interface name in my opinion

Reply:
IKittyCoreTokenTransfer was the updated name.